app/home/user-settings/controller.js

```diff
@@ -0,0 +1,60 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class HomeUserSettingsController extends Controller {
+  @service store;
+  @service router;
+
+  @tracked username;
+  @tracked email;
+  @tracked info;
+  @tracked password;
+  @tracked photoURL;
+  @tracked photo = this.model.photoURL;
+
+  @action
+  onUsernameChange({ target: { value } }) {
+    this.model.username = value;
+  }
+
+  @action
+  onEmailChange({ target: { value } }) {
+    this.model.email = value;
+  }
+
+  @action
+  onInfoChange({ target: { value } }) {
+    this.model.info = value;
+  }
+
+  @action
+  onPasswordChange({ target: { value } }) {
+    this.model.password = value;
+  }
+
+  @action
+  onPhotoURLChange({ target: { value } }) {
+    this.model.photoURL = value;
+  }
```

Comment:
These actions are almost identical. Consider migrate to one, common action `onPropetyChange` or similar which take model key and override it value